index.html

```diff
@@ -0,0 +1,105 @@
+                target="_blank"
+              >
+                <svg
+                  width="20"
+                  height="20"
+                  viewBox="0 0 20 20"
+                  fill="none"
+                  xmlns="http://www.w3.org/2000/svg"
+                >
+                  <path
+                    d="M13.6448 3.03455C10.1561 3.03455 7.28906 5.86213 7.28906 9.37937V19.3104C7.28906 19.6897 7.59994 20.0001 7.97991 20.0001H11.5723C11.9523 20.0001 12.2632 19.6897 12.2632 19.3104V9.65524C12.2632 8.89661 12.8849 8.27592 13.6448 8.27592C14.4048 8.27592 15.0265 8.89661 15.0265 9.65524V19.3104C15.0265 19.6897 15.3374 20.0001 15.7174 20.0001H19.3098C19.6898 20.0001 20.0006 19.6897 20.0006 19.3104V9.37937C20.0006 5.89662 17.1682 3.03455 13.6448 3.03455Z"
+                    fill="#505F79"
+                  />
+                  <path
+                    d="M2.48705 0C1.10535 0 0 1.10345 0 2.48276C0 3.86207 1.10535 4.96552 2.48705 4.96552C3.86874 4.96552 4.97409 3.86207 4.97409 2.48276C4.97409 1.10345 3.86874 0 2.48705 0Z"
+                    fill="#505F79"
+                  />
+                  <path
+                    d="M4.28325 6.27588H0.690846C0.310881 6.27588 0 6.58622 0 6.96553V19.3104C0 19.6897 0.310881 20 0.690846 20H4.28325C4.66321 20 4.97409 19.6897 4.97409 19.3104V6.96553C4.97409 6.58622 4.66321 6.27588 4.28325 6.27588Z"
+                    fill="#505F79"
+                  />
+                </svg>
+              </a>
+            </li>
+            <li>
+              <a href="https://github.com/chepkok3" target="_blank">
+                <svg
+                  width="18"
+                  height="20"
+                  viewBox="0 0 18 20"
+                  fill="none"
+                  xmlns="http://www.w3.org/2000/svg"
+                >
+                  <path
+                    d="M17.9852 7.46114C17.9852 6.32124 17.6192 5.25043 16.9205 4.31779C17.1867 3.41969 17.2532 2.27979 17.1867 0.967185C17.1534 0.414508 16.7209 0 16.1885 0C15.8891 0 13.3604 0.0345423 11.8965 1.38169C10.6321 1.1399 9.30128 1.1399 8.00368 1.38169C6.573 0.0345423 4.04435 0 3.71163 0C3.17928 0 2.74675 0.414508 2.71348 0.967185C2.61366 2.27979 2.71348 3.41969 2.97965 4.31779C2.28095 5.28497 1.91496 6.35579 1.91496 7.46114C1.91496 9.8791 3.71163 12.0553 6.43991 13.0915C6.3401 13.2988 6.27355 13.5406 6.20701 13.7824C3.31237 13.4715 1.94823 10.7427 1.88169 10.639C1.64878 10.1209 1.04989 9.91364 0.550818 10.19C0.0517425 10.4318 -0.147888 11.0535 0.118286 11.5717C0.184829 11.7444 2.01477 15.4404 6.04065 15.8549V18.9637C6.04065 19.5509 6.47319 20 7.0388 20H12.8614C13.427 20 13.8595 19.5509 13.8595 18.9637V14.8532C13.8595 14.2314 13.7264 13.6442 13.4935 13.1261C16.1885 12.0553 17.9852 9.91364 17.9852 7.46114Z"
+                    fill="#505F79"
+                  />
+                </svg>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </section>
```

Comment:
- Good job making some suggested changes by the previous reviewer, but there are still some differences you need to work on. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

- You have not correctly placed the background images, please fix this 🙏 
- The navigation bar in your design is centered in the middle, please reduce the margin between the navbar and outer borders 
- The social links in the design template are 5, but I can only see 3 in your design. please fix this 😄 

Your design:
![qass](https://user-images.githubusercontent.com/73607512/205269685-b76b0b72-2bb2-44f9-a8be-5339f9122e37.PNG)

Figma:
![uuhhu](https://user-images.githubusercontent.com/73607512/205269664-7bd50ad2-5486-4fe7-a13e-2d0bae353b83.PNG)

Reply:
@mckent05 I made the changes and requested a review but it was sent back with a message saying my project does not have linters. I don't understand why that is yet when I make a pull request the linters run and make the changes. Please let me know if I can get help or a guidance since I have the linters set but been told they are not.